src/test/fakes/faker.ts

```diff
@@ -0,0 +1,750 @@
+import {
+    CommPkg,
+    McPkgPreview,
+    Person,
+    PunchItemSavedSearchResult,
+    SavedSearch,
+    ChecklistPreview,
+    PunchPreview,
+    CheckItem,
+    LoopTag,
+    CustomCheckItem,
+    PunchPriority,
+    NewPunch,
+    PunchItem,
+    Attachment,
+    ChecklistDetails,
+    TagDetails,
+    AdditionalTagField,
+    Tag,
+    ChecklistResponse,
+    PunchCategory,
+    PunchType,
+    PunchOrganization,
+    PunchSort,
+    Plant,
+    Project,
+} from './../../services/apiTypes';
+import { faker } from '@faker-js/faker';
+import * as apiTypes from '../../services/apiTypes';
+import {
+    ChecklistSavedSearchResult,
+    PoPreview,
+    TagPreview,
+    WoPreview,
+} from '../../services/apiTypes';
+import * as FakerDataLists from './fakerDataLists';
+import FakerRandomEnum from './fakerRandomEnum';
+import { FakerSyntax } from './fakerSyntax';
+
+export const TagSyntax = 'XXX-AA-XXXX-A-A-XX';
+/**
+ * One Alpha: [a-zA-Z]{1}
+ * Two Alpha: [a-zA-Z]{2}
+ * 'n' Alpha: [a-zA-Z]{n}
+ *
+ * One number: [0-9]{1}
+ * Two number: [0-9]{2}
+ * 'n' number: [0-9]{n}
+ */
+export const TagSyntaxRegex = new RegExp(
+    '^[0-9]{3}-[a-zA-Z]{2}-[0-9]{4}-[a-zA-Z]{1}-[a-zA-Z]{1}-[0-9]{2}'
+);
+export const PurchaseOrderNo = 'XXX-XXX-AA-X';
+
+const TagFunctionCodeSyntax = 'XXX-AAA-XXX';
+const SystemCodeSyntax = 'XXX-XXX-XXX';
+
+export const FakerId = (): number =>
+    faker.datatype.number({ min: 1, max: 999999999 });
+
+export const FakerIdWithIdCheck = (reservedIds: ReservedIds): number => {
+    let id = FakerId();
+    while (reservedIds.ids?.includes(id)) {
+        id = FakerId();
+    }
+    return id;
+};
+export const FakerTitle = (): string => faker.lorem.paragraph(1);
```

Comment:
Hadde det vært mulig å splitte dette i to filer? En fil for disse linjene, faking av en verdi, og en egen fil for faking av ulike api typer? Tror det hadde vært litt ryddigere. Du har jo splittet ut litt av mer kompliserte fakes allerede, men de som ikke trenger en fil alene kunne ha blitt samlet i en fil.

Reply:
Enig!